src/main/scala/firrtl/Emitter.scala

```diff
@@ -242,3 +247,9 @@ case class EmittedVerilogCircuitAnnotation(value: EmittedVerilogCircuit)
     extends EmittedCircuitAnnotation[EmittedVerilogCircuit]
 case class EmittedVerilogModuleAnnotation(value: EmittedVerilogModule)
     extends EmittedModuleAnnotation[EmittedVerilogModule]
+
+final case class EmittedRtlilCircuit(name: String, value: String, outputSuffix: String) extends EmittedCircuit
```

Comment:
please move these classes into the `firrtl.backends.experimental.rtlil` package similar to how we did it for the SMT backend : https://github.com/chipsalliance/firrtl/blob/a921e1230b389be87d993f3016cb46174b1ebfad/src/main/scala/firrtl/backends/experimental/smt/SMTEmitter.scala#L46